README.md

```diff
@@ -50,6 +50,11 @@ The host and port through which Varnish will accept admin requests (like purge a
 
 How Varnish stores cache entries (this is passed in as the argument for `-s`). If you want to use in-memory storage, change to something like `malloc,256M`. Please read Varnish's [Getting Started guide](https://www.varnish-software.com/static/book/Getting_started.html) for more information.
 
+    varnish_pidfile: false
```

Comment:
Can you combine these two vars into one? Basically, default it to:

```
varnish_pidfile: /run/varnishd.pid
```

Then in the templates you can still just do `{% if varnish_pidfile %}` since Python interprets an empty string `""` as `false`. In the comment here in the readme, put "Set to an empty string if you don't want to use a PID file."